open-api/rest-catalog-open-api.yaml

```diff
@@ -173,11 +173,7 @@ paths:
         400:
           $ref: '#/components/responses/OAuthErrorResponse'
         401:
-          $ref: '#/components/responses/UnauthorizedResponse'
-        503:
-          $ref: '#/components/responses/ServiceUnavailableResponse'
-        5XX:
-          $ref: '#/components/responses/ServerErrorResponse'
+          $ref: '#/components/responses/OAuthErrorResponse'
```

Comment:
I think we still want to allow at least a 5XX. That could happen if the OAuth2 endpoint is behind a load balancer or router and the back-end isn't available.

Reply:
I wasn't sure what format we'd use, the OAuth spec doesn't say anything about 500s. I'll put it back as an OAuthErrorResponse format if that makes sense.